schema.json

```
@@ -102,6 +102,10 @@
       "default": false,
       "type": "boolean"
     },
+    "commitish": {
+      "default": "",
+      "type": "string"
+    },
```

I forgot we actually are able to generate this using `yarn generate-schema` and this is what it generates based on the `/lib/schema.js`
```suggestion
    "commitish": {
      "anyOf": [
        {
          "type": "string",
          "enum": [""]
        },
        {
          "default": "",
          "type": "string"
        }
      ]
    },
```